frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -41,6 +41,11 @@ const CampOverviewPage = (): JSX.Element => {
   });
   const numSessions = camp.campSessions?.length;
 
+  const [refetch, setRefetch] = useState<boolean>(true);
```

Comment:
I'm only adding this to manually trigger a refetch after the camper has been deleted so that the camper doesn't remain in the table, which creates quite a bit of prop drilling. Any alternatives?